wordpress-mobile/WordPress-iOS
# Extract Stats Remote Services into WordPressKit dynamic framework

Issue #7144 

This PR merges the results of PR #7146 plus extraction of the stats remote services into WordPressKit. WordPressKit is a dynamic framework that will eventually house all remote services and related data models. This is the first major re-structuring of the networking stack to support the iOS networking rearchitecture project.

To test:
1. Close Xcode.
2. Delete your derived data (to be safe).
3. `pod install`.
4. Open the workspace.
5. Build & run unit tests.
6. Build & run the app. Make sure stats works properly.

Needs review: @elibud, @frosty, @bummytime
